client/src/assets/resonance.svg

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I know that this file is auto generated, but it would look great if you format it :).
Be careful importing this as React component, because the class names might be injected to your main page.
Thus, you might face conflicts if you import a different file with the same class names (like `st4`).
Maybe it would be a good idea to rework this file and get rid of the classes. 

In addition, there might be some unused properties. 
Same issue in other svg files.

Reply:
Thanks for the hint. I cleaned up the two svg files and it really is better that way!